.github/workflows/release.yml

```diff
@@ -0,0 +1,82 @@
+name: Docker Release
+
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+
+jobs:
+  release_config_builder:
+    name: Release Cass Config Builder
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+      - name: Login to DockerHub
+        run: echo "${{ secrets.DOCKER_PASSWORD }}" | docker login -u "${{ secrets.DOCKER_USERNAME }}" --password-stdin
+      - name: Setup Buildx
+        id: buildx
+        uses: crazy-max/ghaction-docker-buildx@v3
+      - name: Cache Docker layers
+        uses: actions/cache@v2
+        id: cache
+        with:
+          path: /tmp/.buildx-cache
+          key: ${{ runner.os }}-buildx-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-buildx-
+      - name: Publish to Dockerhub
+        run: |
+          GITHUB_REPO_URL="https://github.com/${{ github.repository }}"
+
+          VERSION_NUMBER="$(cat version.txt | tr -d '[:space:]')"
+          RELEASE_VERSION="${VERSION_NUMBER}"
+
+          DOCKERHUB_REPOSITORY="datastax/cass-config-builder"
+
+          # Make sure the version number of the project aligns with
+          # the tag that we have to prevent confusion.
+          GIT_TAG="${GITHUB_REF##*/}"
+          if ! [ "v${VERSION_NUMBER}" = "${GIT_TAG}" ]; then
+            echo "Git tag $GIT_TAG does not align with version number ${VERSION_NUMBER} in version.txt"
+            exit 1
+          fi
+
+          DOCKERHUB_TAGS=(--tag "${DOCKERHUB_REPOSITORY}:${RELEASE_VERSION}")
+          DOCKERHUB_UBI_TAGS=(--tag "${DOCKERHUB_REPOSITORY}:${RELEASE_VERSION}-ubi7")
+
+          LABELS=(
+            --label "release=$RELEASE_VERSION"
+            --label "org.label-schema.schema-version=1.0"
+            --label "org.label-schema.vcs-ref=$GITHUB_SHA"
+            --label "org.label-schema.vcs-url=$GITHUB_REPO_URL"
+            --label "org.label-schema.version=$RELEASE_VERSION"
+          )
```

Comment:
This information all seemed useful to include in the docker image and I found label-schema (http://label-schema.org/) which attempts to offer some standardization around it. I have no idea how widely it's used though.

Reply:
docker image labels are great, but k8s seems to be pretending that they provide no value and, at least a few months ago, had no way to inspect or expose them inside of k8s.  That being said, I totally support us setting docker image labels.

Reply:
Yeah, not really looking at using the labels in k8s per se. The version and sha are useful when generating a release as we can ensure we are pushing the correct image to dockerhub (not really relevant for config builder at the moment as we have to rebuild for a release).